system/gatekeeper/tests/suite.yaml

```
@@ -266,6 +266,20 @@ tests:
     assertions:
     - violations: no
 
+- name: region-value-mismatch
+  template: rendered-chart/gatekeeper/templates/constrainttemplate-region-value-mismatch.yaml
+  constraint: rendered-chart/gatekeeper-config/templates/constraint-region-value-mismatch.yaml
+  cases:
+  - name: helm-release-region-mismatch
+    object: fixtures/region-value-mismatch/helm-release-region-mismatch.out.yaml
+    assertions:
+    - violations:
+      message: '.Values.global.region does not match the deployed region'
```

It would be nice if this message included the specific region values encountered.
```suggestion
      message: '.Values.global.region does not match the deployed region: expected qa-de-1, but found qa-de-9'
```